sktime/transformations/series/summarize.py

```diff
@@ -750,3 +750,140 @@ def get_test_params(cls, parameter_set="default"):
         params4 = {"summary_function": None, "quantiles": (0.1, 0.2, 0.25)}
 
         return [params1, params2, params3, params4]
+
+
+class SplitterSummarizer(BaseTransformer):
+    """Create summary values of a time series' splits.
+
+    A series-to-series transformer that applies a series-to-primitives transformer
```

Comment:
Good description - I would add a little more detail, e.g., what are the resulting indices, and what is in the i-th row, etc.

Reply:
Added